config/src/projects/zkswapv2.ts

```diff
@@ -0,0 +1,303 @@
+import { RISK } from './common'
+import { Project } from './types'
+
+export const zkswapv2: Project = {
+  name: 'ZKSwap (v2)',
+  slug: 'zkswapv2',
+  bridges: [
+    {
+      address: '0x6dE5bDC580f55Bc9dAcaFCB67b91674040A247e3',
+      sinceBlock: 12810001,
+      tokens: [
```

Comment:
Please update this list to:
```
1INCH
AAVE
ETH
GT
LRC
OKB
SNX
SUSHI
UNI
USDC
USDT
WBTC
ZKS
```

I have run the code and verified that the other tokens are not stored in the contract.

Reply:
I have a quesiton. Once someone deposited a token which is not in the above, shoud I create another PR to update this? ZKSwap v2 now supports listing tokens by user.

Reply:
Currently that is the case. In the future we plan to just check all tokens against all bridges, but because of the current backend architecture it would be very inefficient.